src/dialogs/filedatedialog.cpp

```diff
@@ -113,7 +115,8 @@ void FileDateDialog::populateDateOptions(Mlt::Producer *producer)
     }
 
     // Add metadata dates
-    Mlt::Producer tmpProducer( *(producer->profile()), "avformat", resource.toUtf8().constData() );
+    Mlt::Profile profile(mlt_profile_clone(producer->get_profile()));
+    Mlt::Producer tmpProducer( profile, "avformat", resource.toUtf8().constData() );
```

Comment:
Why not use `MLT.profile()` here like we do so many other places?
There is not supposed to be spaces inside the parenthesis.

Reply:
Not a compelling reason - but so much depends on mltcontroller.h. If I change one small thing in mltcontroller.h, the whole application has to recompile. I think we should prefer MLT.profile() over Mlt::Service::profile() in all cases.

Reply:
> There is not supposed to be spaces inside the parenthesis.

done